Cargo.toml

```diff
@@ -23,7 +23,9 @@ hex = { version = "0.2", package = "hex-conservative" }
 log = "^0.4"
 minreq = { version = "2.11.0", features = ["json-using-serde"], optional = true }
 reqwest = { version = "0.11",  features = ["json"], default-features = false, optional = true }
-async-std = { version = "1.13.0", optional = true }
+
+# default async runtime
+tokio = { version = "1.38", features = ["time"], optional = true }
```

Comment:
Would you mind relaxing the Tokio requirement to `1.36` or lower? `tokio::time::sleep` has been around since Tokio at least `1.0.0`.

(Asking because we happen to use 1.36 and Tokio is a pain to update because we maintain a Tokio patchset for SGX)

Reply:
I don't see much problem with this suggestion, we could use as in BDK: `tokio = { version = "1", features = ["time"], optional = true }`. Any thoughts @ValuedMammal ?

Reply:
@oleonardolima @MaxFangX changed it to "1"